orc8r/tools/packer/scripts/upgrade-ovs-patch-1.1.20210618.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+################################################################################
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+set -euo pipefail
+set -x
```

Comment:
If this is for debugging only, it should be removed. Otherwise, it should probably be combined with the above set, e.g. `-xeuo`.

Reply:
is it best practice to keep it separated, since `set -euo pipefail` is expected and well known. And it makes it explicit.